### PR TITLE
Initial Organization Setup

### PR DESCRIPTION
**Pull request by martinheidegger**
Added a deeper description of the organization structure. This is a draft, what do you think?

### CODE REVIEW

**Review by hccampos — APPROVED** [review]
Looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+- **developers:** Developers have write access to all repositories.
+
+We generally recommend that any contributor with two PR's join our organization.
+
+## History
+There has been a lot of prior work to the work, displaying pointcloud data. This project original started as fork of [potree](https://github.com/potree/potree) but also draws inspiration from [plasio](https://github.com/hobu/plasio.js),
+[entwine](https://github.com/connormanning/entwine) and [Cesium](https://cesiumjs.org/).
+
+## License
+All projects in this group are to be published under the [MIT](https://en.wikipedia.org/wiki/MIT_License) License.
```
Comment (martinheidegger) [inline README.md]: @mschuetz just pointed out that Potree is using the [BSD License](https://github.com/potree/potree/blob/develop/LICENSE), which means that any code would need to be either BSD or completely rewritten.

**Review by martinheidegger — COMMENTED** [review]
@pnext/admin move to BSD?

File: README.md (hunk 1 above)
Comment (dkastl) [reply README.md]: Any reason, that it shouldn't be BSD?
Whether it's BSD or MIT, they are both permissive licenses.

Comment (martinheidegger) [reply README.md]: I added MIT out of simple habit, was a mistake but I just wanted @hccampos's input

Comment (hccampos) [reply README.md]: IANAL but I think they end up being almost the same and I think they are also compatible, so either way is good.

**Pull request merged**